sdk/appconfiguration/azure-data-appconfiguration/samples/appconfig_basic_operation.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @brief This sample provides the code implementation to use the App Configuration client SDK for
+ * C++ to get one or more configuration settings.
+ */
+
+#include <azure/data/appconfiguration.hpp>
+#include <azure/identity.hpp>
+
+#include <iostream>
+
+using namespace Azure::Data::AppConfiguration;
+using namespace Azure::Identity;
+
+int main()
+{
+  try
+  {
+    std::string url = "https://<your-appconfig-name>.azconfig.io";
+    auto credential = std::make_shared<Azure::Identity::DefaultAzureCredential>();
+
+    // create client
+    ConfigurationClient configurationClient(url, credential);
+
+    Azure::Response<GetKeyValueResult> response = configurationClient.GetKeyValue("key", "accept");
```

Comment:
Known issue: The `accept` parameter shouldn't be required.
I am not sure why the emitter is adding this, so requires investigation.

Also, we intend to rename this to `GetConfigurationSetting` to align with other languages.